src/main/java/graphql/execution/ValuesResolver.java

```diff
@@ -223,6 +223,16 @@ public static Object valueToInternalValue(InputValueWithState inputValueWithStat
         return assertShouldNeverHappen("unexpected value state " + inputValueWithState);
     }
 
+
+    @Nullable
+    @SuppressWarnings("unchecked")
+    public static <T> T getInputValueImpl(GraphQLInputType inputType, InputValueWithState inputValue) {
```

Comment:
I am not sure this is what we want always: sometimes not set is different from null.